### ISSUE TITLE
KerasRnnUtils.getRecurrentDropout cannot handle parameters as Integer (Keras import)

### ISSUE THREAD

**Issue by GrgrLmml** [deeplearning4j/deeplearning4j#4568]
#### Issue Description

For LSTM layers Keras serialize dropout parameters as integers. Keras importer fails because of a ClassCastException

- relevant python code:

      model = Sequential()
      ...
      model.add(LSTM(500, kernel_initializer='VarianceScaling', recurrent_initializer='VarianceScaling')) 
      ...
      json_string = model.to_json()
      text_file = open("/tmp/model.json", "w")
      text_file.write(json_string)
      text_file.close()

- relevant json snipet:

        "dropout": 0,
        "recurrent_dropout": 0,

- relevant java code:

      MultiLayerConfiguration modelConf = KerasModelImport.importKerasSequentialConfiguration("/tmp/model.json");

- exception:
Exception in thread "main" java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Double
	at org.deeplearning4j.nn.modelimport.keras.layers.recurrent.KerasRnnUtils.getRecurrentDropout(KerasRnnUtils.java:48)
	at org.deeplearning4j.nn.modelimport.keras.layers.recurrent.KerasLstm.<init>(KerasLstm.java:138)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:197)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:99)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:60)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildSequential(KerasModelBuilder.java:128)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialConfiguration(KerasModelImport.java:306)

#### Workaround
changing dropout / recurrent_dropout in model.json to:

        "dropout": 0.0,
        "recurrent_dropout": 0.0,

solves the issue.

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version: 0.9.2-SNAPSHOT

**Comment by maxpumperla** [comment]
@GrgrLmml thanks for reporting this. easy enough to fix. :)

**Comment by maxpumperla** [comment]
works on master now (with test) and should be in snapshots soon enough.
